game/src/game/select_song.rs

```diff
@@ -40,6 +41,13 @@ pub(crate) fn select_song(
     songs: &Vec<GameSong>,
 ) -> SongSelectionResult {
     let texture_creator = common_context.canvas.texture_creator();
+    let mut janggu_state = JangguStateWithTick::new();
+
+    let mut hit_left = false;
+    let mut hit_right = false;
+    let mut last_left_hit_time = Instant::now();
+    let mut last_right_hit_time = Instant::now();
+    let hit_both_side_time_delay = 100; // ms
```

Comment:
When I tested this with the janggu, the delay wasn't long enough. Based on my tests, 200ms seemed more appropriate. However, since it can vary from person to person, we can test it together later and then decide.